ILSpy.ReadyToRun/Properties/Resources.zh-Hans.resx

```diff
@@ -129,4 +129,7 @@
   <data name="ShowUnwindInfo" xml:space="preserve">
     <value>显示展开信息</value>
   </data>
+  <data name="ShowGCInfo" xml:space="preserve">
+    <value>?</value>
```

Comment:
Don't know how to get a proper value here

Reply:
Initially you can leave the chinese version unmodified, it'll fall back to the english resources for missing strings.